rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -310,15 +310,22 @@ In an empty `TransactionPayload` response message the transaction reference MUST
 
 See Appendix A.3 for the reasoning behind the empty `TransactionPayload` response.
 
-8. Errors
+## 8. Diagnostics
```

Comment:
We should clarify the contents of the `Diagnostics` message. RFC005 did this by including a reference to the protobuf definition, but I don't think this is in RFC017 yet.

Reply:
Added